packages/backend/server.go

```diff
@@ -37,9 +40,19 @@ func NewServerManager (ctx context.Context, conns ...*dbus.Conn) (*Manager, erro
         return nil, status.Errorf(codes.Internal, "Failed to create user service: %s", err)
     }
 
+    c := &http.Client{
+        Transport: &http.Transport{
+            DialContext: func(ctx context.Context, network, addr string) (net.Conn, error) {
+                return (&net.Dialer{}).DialContext(ctx, "unix", "/run/snapd.socket")
```

Comment:
This is something I hit with the prompting client when I was getting it packaged up as a snap: there's a different socket we need to connect to in that case. [This](https://github.com/canonical/aa-prompting-test/blob/main/aa-prompt-client/src/snapd_client.rs#L116-L120) was the simplest way to check for which socket we should use that I could come up with (the `snapd` folks also suggested that this was the way to go).

Not a blocker for merging but something we'll need to remember to take a look at before we package as a snap.